### PR TITLE
Add puppeteer PDF renderer

### PR DESCRIPTION
**Pull request by quantum5**

### CODE REVIEW

File: judge/pdf_problems.py [hunk 1]
```diff
@@ -32,6 +42,12 @@ def load(self, file, source):
             target.write(source.read())
 
     def make(self, debug=False):
+        self._make(debug)
+
+        if self.title and HAS_EXIFTOOL:
+            subprocess.call([EXIFTOOL, '-Title=%s' % (self.title,), self.pdffile])
```
Comment (Xyene) [inline judge/pdf_problems.py]: The lack of logging here has me concerned.

Comment (quantum5) [reply judge/pdf_problems.py]: I was thinking that it's unimportant since nothing really happens if `exiftool` dies, but yea, probably should log it...

Comment (quantum5) [reply judge/pdf_problems.py]: Added logging for `exiftool` errors.

**Comment by quantum5** [comment]
We need... `libasound2 libxss1 libxtst6`.

**Pull request merged**